### PR TITLE
completed css positioning-Bo Mercado

### PR DESCRIPTION
**Pull request by rbomercado**
image

image
fixed position

### CODE REVIEW

File: css-positioning/absolute.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+.meme > img {
+  width: 100%;
+}
+
+.emoji {
+  font-size: 3rem;
+
+  /* your code here */
+  position: absolute;
+  left: 230px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:19]: Oh no, this is a **_massive_** movement. Remember what I said when introducing the exercise, you should try to use small movements from the side closest to where you want the content to be. I would suggest taking another look at this piece of the instructions. Development requires **_very careful reading_**.
image

File: css-positioning/absolute.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  margin-bottom: 1rem;
+  font-size: 2rem;
+  font-family: Anton, sans-serif;
+  text-align: center;
+  color: white;
+  -webkit-text-stroke: 1px black;
+  text-transform: uppercase;
+  position: absolute;
+  width: 300px;
+  top: 175px;
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:32]: Same thing here, this movement is massive.

File: css-positioning/absolute.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+  /* your code here */
+}
+
+.overlay {
+  background: linear-gradient(transparent, rgb(0 0 0 / 75%));
+
+  /* your code here */
```
Comment (codingcodymiller) [inline css-positioning/absolute.css:40]: Did you forget about the overlay?

File: css-positioning/relative.css [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+.pokemon > img {
+  width: 150px;
+
+  /* your code here */
+}
+
+#bulbasaur {
+  /* your code here */
+  position: relative;
+  right: 320px;
```
Comment (codingcodymiller) [inline css-positioning/relative.css:16]: I told you when introducing the exercise that you don't need to worry about moving these left or right at all 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by rbomercado** [comment]
image

image

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**